### PR TITLE
Clean up desktop's index.html

### PR DESCRIPTION
**Pull request by asaaki**
Initially I wanted to delete only the duplicated `<html>` opening tag, but then decided to make it a fully valid HTML document.

Passes https://validator.w3.org/nu/#textarea check.

### CODE REVIEW

File: packages/desktop/src/index.html [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-</body>
-
-<script>
-    import("./index.js").then(function (module) {
-        module.main();
-    });
-</script>
-
+    <head>
+        <title>Dioxus app</title>
```
Comment (asaaki) [inline packages/desktop/src/index.html:4]: This is not strictly necessary for dioxus, but turns this file into a fully valid document.
It get's overwritten by dioxus or the user's application configuration anyway.

Comment (mrxiaozhuox) [reply packages/desktop/src/index.html:4]: I don't know should we add this `title` tag. but maybe it doesn't matter.

**Comment by mrxiaozhuox** [comment]
looks good, you have format the HTML code.

**Comment by jkelleyrtp** [comment]
Thank you!

**Pull request merged**